pkg/render/renderer_impl.go

```diff
@@ -71,11 +70,13 @@ type Renderer struct {
 	templatesRoot string
 }
 
+var _ Renderer = (*Impl)(nil) // ensure interface satisfied
+
 // New creates a new renderer with the given details.
-func New(ctx context.Context, root string, debug bool) (*Renderer, error) {
+func New(ctx context.Context, root string, debug bool) (*Impl, error) {
```

Comment:
don't you want the return type to be `Renderer`?

Reply:
+1

Reply:
Done